test/unit/verify/test_models.py

```diff
@@ -25,6 +25,7 @@
 
 
 class TestVerificationMaterials:
+    @pytest.mark.xfail
```

Comment:
> At the moment these tests load the commited bundle.txt.sigstore which does not contain the checkpoint field. This seems to point to backwards-compatibility concerns...

That's probably because it's not included in our own bundle generation code 🙂 -- it's an optional field in the protobuf definition, which means that it was probably an oversight on my part. 

If the current bundles are missing checkpoints, we should fix the bundle generation code and check in some new (additional) bundles that contain checkpoints on their tlog entries. The current ones should stay and be renamed to indicate that they lack checkpoints, since they serve as a good behavioral test for handling that case.